### ISSUE TITLE
[ Programmatic access-Howdy-for-pms]: Tooltip is not provided to “search “text box.

### ISSUE THREAD

**Issue by v-gasikk** [howdyai/howdy-website#17]
**User Experience**
Proper relationship should be provided to all control so that user, with low vision, getting interact with every element consistently.

**Test Environment**
OS: Win 10 Version 1809 (OS Build 17763.316)
URL: https://hcl-szz1925.howdy.ai/dashboard
Browser: Edge
 
**Repro Steps**
Step 1: Open the URL: https://hcl-szz1925.howdy.ai/dashboard
Step 2: Navigate to "Dashboard" tab and select it.
Step 3: Navigate to "Howdy for Your PM" link and select it.
Step 4: "Howdy for Your PM" page gets opened.
Step 5: Navigate to search text box 
Step 6: Observe the issue 

**Actual Result**
Tooltip is not provided to “search “text box howdy for pms in the page.

**Expected Result**
Tooltip should be provided for search text box in the howdy for pms page.

**Note** Screen reader is also not reading the search results and search information.
Please refer the attachment:MAS1.3.3_Search results.wmv

**Mas Reference**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1823da5a-e8fb-4e63-b881-7eccb312c358}
[MAS1.3.3_Search text box.zip](https://github.com/howdyai/howdy-website/files/2973766/MAS1.3.3_Search.text.box.zip)
![MAS1 3 3_Search text box](https://user-images.githubusercontent.com/48272005/54473802-acef2900-4802-11e9-957e-ed8063dddb38.png)
[MAS1.3.3_Search results.zip](https://github.com/howdyai/howdy-website/files/2973769/MAS1.3.3_Search.results.zip)

**Comment by benbrown** [comment]
This page has been deprecated and removed from the site.

**Comment by v-gasikk** [comment]
Now this page is not available. Closing this issue based on developer comment.
